src/engine/renderer/Material.cpp

```diff
@@ -1870,13 +1870,11 @@ void MaterialSystem::AddDrawCommand( const uint32_t materialID, const uint32_t m
 	}
 
 	cmd.cmd.count = count;
-	cmd.cmd.instanceCount = 1;
```

Comment:
Why are those two fields needed at all? If I grep for `baseVertex` and `instanceCount` I don't see anything that reads them

Reply:
It's used by `glMultiDrawElementsIndirectCountARB()` eventually.